src/Utilities/Routes.js

```diff
@@ -61,7 +63,15 @@ const InsightsRoute = ({ component: Component, title, ...rest }) => {
         <Route
             {...rest}
             render={routeProps => (
-                !hasSystems ? <Upgrade /> : <Component {...routeProps} />
+                !hasSystems ?
+                    <AsynComponent
```

Comment:
Small typo here :smile:

Reply:
Ahh nice catch

Reply:
Actually, this seems to be how its exported, I've changed the name here but the default is like this